### PR TITLE
Release Notes 6.5.0

### PR DESCRIPTION
**Pull request by CarlosFdez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,97 @@
...
+-   (DocSchlock) Add silver to Moonlit Spellgun
+-   (Hydrair) Fix selectors of Curse of Ancestral Meddling
+-   (Intervencion) Fix Solar Rejuvenation healing formula
+-   (rectulo) Fix level for wand of purification
+-   (reyzor1991) Add Note for Envenom Companion
+-   (reyzor1991) Change action cost from 1 to 2 Con for Rit - Spit Venom
+-   (Rigo) Add Strike failure note to Energizing Lattice effect
+-   (Rigo) Add area roll options to several inline links
+-   (Rigo) Add effect for Sixth Pillar Mastery
+-   (Rigo) Add fly speed to owl bear
```
Comment (pedrogrullada) [inline CHANGELOG.md:68]:
```suggestion
-   (Rigo) Add fly speed to Winged Owlbear
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,97 @@
...
+-   (Rigo) Add inline healing rolls to Vital Beacon
+-   (Rigo) Add labeled badge to Sewer Ooze's Filth Wave effect
+-   (Rigo) Automate Backfire Mantle, Blessed Swiftness, Dazzling Rosary, Dragon Throat Scale, Necrotic Cap, Wemmuth Trinket, and Pentagonal Seventh Prison
+-   (Rigo) Clean up effects for Healer's Gel and Serene Mutagen
```
Comment (pedrogrullada) [inline CHANGELOG.md:72]:
```suggestion
-   (Rigo) Clean up Serene Mutagen effects
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,97 @@
...
+-   (Rigo) Condense Aura of Faith effects and enable Champion's Aura by default
+-   (Rigo) Condense Elixir of Life effects and add Life Shot effect
```
Comment (pedrogrullada) [inline CHANGELOG.md:74]:
```suggestion
-   (Rigo) Condense Elixir of Life and Healer's Gel effects and add Life Shot effect
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,97 @@
...
+-   (stwlam) Refrain from broadcasting drag measurement of GM-hidden tokens
+-   (Supe) Make effects panel scrollable when overflowing
+-   (Supe) Recharge turn and round abilities as the encounter changes turns
+-   (Supe) Reduce remaining action uses when clicking the use button
+-   (Tikael) Allow alternative handwraps to function like Handwraps of Mighty Blows
+
+## Bug Fixes
+
+-   (Dire Weasel, Supe, stwlam) Fix missing compendium sources in several items
+-   (Stefan van Bodegraven) Fix Crash/Freeze when missing @Damage closing bracket
```
Comment (TikaelSol) [inline CHANGELOG.md:16]:
```suggestion
-   (Stefan van Bodegraven) Fix Crash/Freeze when missing inline damage closing bracket
```
We've pinged @ damage enough on here I think

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,97 @@
...
+-   (Supe) Fix refreshing items embedded in compendium actors
+
+## Data Updates
+
+-   (Ambrose, Dire Weasel) Update more item descriptions and durations for remaster
+-   (7H3LaughingMan) Implement Monastic Weaponry's Criticial Specialization
+-   (Abbadon) Brush up multiple effects
+-   (Ambrose) Add Divine Access class feature to Oracle journal entry
+-   (Ambrose) Add Fortitude Save to Envenom Companion spell
+-   (Ambrose) Add Gelid Shark to archetype journal
```
Comment (TikaelSol) [inline CHANGELOG.md:29]:
```suggestion
-   (Ambrose) Add Gelid Shard archetype to archetype journal
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,97 @@
...
+-   (Ambrose) Add Volley 60 feet trait
+-   (Ambrose) Allow strike sanctification to apply to champion archetype
+-   (Ambrose) Automate "Come and Get Me"
+-   (Ambrose) Brush up select Bestiary Effects
+-   (Ambrose) Condense Antidote, Antiplague, and Blood Booster item effects
+-   (Ambrose) Fix Fire Shield
+-   (Ambrose) Redirect more Bestiary actors to Monster Core equivalents
+-   (Ambrose) Tag more feats that grant ancestral spells
+-   (Ambrose) Update GM Screen journal and add Traits page
```
Comment (TikaelSol) [inline CHANGELOG.md:38]:
```suggestion
-   (Ambrose) Rewrite GM Screen journal
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,97 @@
...
+-   (CrackJackFlood) Condense Tail Lash Effects
+-   (CrackJackFlood) Fix Soulbound Ruin statblock
+-   (Daomephsta) Fix inline checks for Paradox Engine
+-   (Dire Weasel) Add automation for Bomber's Eye Elixir, Hard to Target, Stag's Helm, and Weapon Siphon
+-   (Dire Weasel) Add template to Bloodsiphon's Death Burst
+-   (Dire Weasel) Brush up Old Thornbarker
+-   (Dire Weasel) Condense effects for Mudrock Snare
+-   (Dire Weasel) Condense effects for Treat Disease and Treat Poison
+-   (Dire Weasel) Fix Masquerade Scarf, Adamantine Body, and various item descriptions
+-   (Dire Weasel) Fix some token names
```
Comment (TikaelSol) [inline CHANGELOG.md:55]: Can lose this one

Comment (TikaelSol) [reply CHANGELOG.md:55]: The token name one specifically

**Pull request merged**